src/stream/src/executor/managed_state/join/mod.rs

```diff
@@ -300,39 +295,43 @@ impl<K: HashKey, S: StateStore> JoinHashMap<K, S> {
             table: degree_table,
         };
 
-        Self {
-            inner: EvictableHashMap::with_hasher_in(
-                target_cap,
+        let cache = if let Some(lru_manager) = lru_manager {
+            ExecutorCache::Managed(
+                lru_manager.create_cache_with_hasher_in(PrecomputedBuildHasher, alloc),
```

Comment:
`create_cache_with_hasher_in` seems to be only used here. So, does it make sense to pass a customized allocator to HashJoin now? I think it was an early-stage memory management approach which has been replaced by this one.  cc. @TennyZhuang